babel-macro/src/macro.ts

```diff
@@ -24,15 +24,48 @@ interface MacrosBabel {
   types: typeof babelTypes;
 }
 
-function findExportedMethods(ast: babelTypes.File, { types: t }: MacrosBabel): string[] {
+function getFunctionName(e: babelTypes.FunctionDeclaration, t: typeof babelTypes): string | undefined {
+  return t.isIdentifier(e.id) ? e.id.name : undefined;
+}
+
+function verifyExportedMember(ast: babelTypes.File, t: typeof babelTypes, funcName: string): void {
```

Comment:
maybe assetExportedMember (since it throws otherwise)

Reply:
changed